docs/excel_import/README.md

```diff
@@ -27,6 +27,9 @@ Excel Column | Content Designer Field| Description
  cardimageurl | Card Image Url | The web address of an image that will appear on the response card
  displaytext1...displaytext(n) | Display Text | The text of the button. Each button text should be in a separate column numbered button1, button2, ...
  buttonvalue...buttonvalue(n) | Button Value| The value of the button. Each button value should be in a separate column numbered button1, button2, ...
+ attributename1...attributename(n) | Session Attribute Name | The name of the session attribute. Each attribute name should be in a separate column numbered attributename1, attributename2, ...
```

Comment:
Could you please provide steps to tests this feature (test case with required assets (e.g., sample inputs)), so our QA can added to our test cases for the coming release.

Reply:
The Excel file  in docs/excel_import/ has been updated with sample session attributes.

Reply:
Thanks